### DISCUSSION TITLE
Cant make it run on Surface Pro 2017 (intel core m3)

### DISCUSSION THREAD

**Discussion by Citoci** [sebanc/brunch#796] [Q&A]
I tried to install something like 10 or 12 different recoveries, but the brunch framework always gets stuck on "ChromeOS rootfs patches are being applied" (I "was patient" for 2 hours ;P). I tested the "rammus" recovery on another laptop with an intel core i5, and it works perfectly there... it seems that the surface is cursed ;(
Does anyone have an idea on how to make it work? Thanks :D

**Comment by sebanc** [comment]
I have this exact same model and brunch works fine on it, could you remove the text "console= vt.global_cursor_default=0 brunch_bootsplash=default" from the kernel command line and post a photo of where it gets stuck ?

**Reply by Citoci** [reply]
Nevermid... I have no explanation for this, but after a couple of days of attempts, it finally booted... And now I cant make that error that I always got before appear again... The really strange thing is that I've always used the same procedure, only changing the recovery image (in fact it just worked well at the first attempt on my other laptop), but trying it on the surface was always leading to the same error in installation (in debug mode the last line printed in output was something like "chromeos-self-reset")
I apologize for the trouble, thank you very much for the answer!
